src/cmd/linuxkit/moby/config.go

```diff
@@ -110,12 +110,13 @@ type ImageConfig struct {
 
 // Runtime is the type of config processed at runtime, not used to build the OCI spec
 type Runtime struct {
-	Cgroups    *[]string      `yaml:"cgroups,omitempty" json:"cgroups,omitempty"`
-	Mounts     *[]specs.Mount `yaml:"mounts,omitempty" json:"mounts,omitempty"`
-	Mkdir      *[]string      `yaml:"mkdir,omitempty" json:"mkdir,omitempty"`
-	Interfaces *[]Interface   `yaml:"interfaces,omitempty,omitempty" json:"interfaces,omitempty"`
-	BindNS     Namespaces     `yaml:"bindNS,omitempty" json:"bindNS,omitempty"`
-	Namespace  *string        `yaml:"namespace,omitempty" json:"namespace,omitempty"`
+	Cgroups     *[]string          `yaml:"cgroups,omitempty" json:"cgroups,omitempty"`
+	Mounts      *[]specs.Mount     `yaml:"mounts,omitempty" json:"mounts,omitempty"`
+	Mkdir       *[]string          `yaml:"mkdir,omitempty" json:"mkdir,omitempty"`
+	Interfaces  *[]Interface       `yaml:"interfaces,omitempty,omitempty" json:"interfaces,omitempty"`
+	BindNS      Namespaces         `yaml:"bindNS,omitempty" json:"bindNS,omitempty"`
+	Namespace   *string            `yaml:"namespace,omitempty" json:"namespace,omitempty"`
+	EnvFromFile *map[string]string `yaml:"envFromFile,omitempty" json:"envFromFile,omitempty"`
```

Comment:
It wasn't very clear that I was supposed to do this, and having a reference to a map also felt a little odd.

Reply:
From memory, they are pointers so they get totally omitted from the generated JSON file if not set, rather than being set to an empty list or map, so the JSON is less verbose. Could add a comment to this effect.